mlflow/utils/databricks_utils.py

```diff
@@ -335,6 +335,57 @@ def get_workspace_url():
         return None
 
 
+def is_mlflowdbfs_available():
+    if not is_in_databricks_runtime():
+        return False
+
+    # allow users to choose not to use mlflowdbfs via an environment variable
+    disable_mlflowdbfs = os.environ.get("DISABLE_MLFLOWDBFS")
+    if disable_mlflowdbfs:
+        return False
+
+    try:
+        spark_session = _get_active_spark_session()
+    except Exception:
+        return False
+
+    if spark_session is None:
+        return False
+
+    from py4j.protocol import Py4JJavaError
+
+    try:
+        spark_session.read.load("mlflowdbfs:///123")
+    except Py4JJavaError as e:
+        if str(e.java_exception) == 'No FileSystem for scheme "mlflowdbfs"':
+            return False
+
+        return True
```

Comment:
We should determine whether or not there's a higher-fidelity way to check for the presence of `mlflowdbfs`. This approach where we perform case-sensitive string matching on exception content is a bit brittle.